e2e/ctl_v3_snapshot_test.go

```diff
@@ -119,3 +126,152 @@ func getSnapshotStatus(cx ctlCtx, fpath string) (snapshotStatus, error) {
 	}
 	return resp, nil
 }
+
+func TestCtlV3SnapshotRestore(t *testing.T) {
```

Comment:
// Issue6361 tests the scenarios:
// (describes what this test checks) 
TestIssue6361